app/admin/page.tsx

```diff
@@ -0,0 +1,104 @@
+'use client';
+
+import { useSession, signIn, signOut } from 'next-auth/react';
+import Button from '@/components/all/Button';
+import Link from 'next/link';
+import { CircleDollarSignIcon, PaperclipIcon, UserIcon } from 'lucide-react';
+
+const AdminPage = () => {
+  const { data: session } = useSession();
+
+  const handleLogin = () =>
+    signIn('google', {
+      callbackUrl: '/admin',
+    });
+
+  const handleLogout = () => signOut({ callbackUrl: '/' });
+
+  const routes = [
+    {
+      title: 'Portfølje',
+      href: '/admin/portfolio',
+      icon: CircleDollarSignIcon,
+    },
+    {
+      title: 'Medlemmer',
+      href: '/admin/members',
+      icon: UserIcon,
+    },
+    {
+      title: 'Søknader',
+      href: '/admin/applications',
+      icon: PaperclipIcon,
+    },
+  ];
+
+  if (!session) {
```

Comment:
Kanskje vise denne logg in siden på alle /admin sider. Kan  leegge til en `layout.tsx` under `/admin`-mappen